tests/test_LookupTable.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/python
+# -*- coding: utf-8  -*-
+import unittest
+import importer.LookupTable as Lt
+
+
+class TestTableConverter(unittest.TestCase):
+
+    def setUp(self):
+        self.converter = Lt.LookupTable("test")
+        self.wikipage = """lorem ipsum\n{| class="wikitable sortable"\n! Name\n! Count\n! Item(s)\n! Notes\n|-\n| kloster\n|\n| {{Q|Q44613}}\n|\n|-\n| pestsäule\n|31\n| {{Q|Q1549521}}\n| \n|}"""
```

Comment:
could you line break this on `\n`'s. It would make it easier to read and it would look more like "familiar" wikitext.
A suggestion is to use single quotes but wrap it all in a bracket to keep sane indentation but avoid `\` line endings.